### PR TITLE
Issue templates REDUX

### PR DESCRIPTION
**Pull request by roobre**
Follow up from #17, leveraging Issue forms.

After review, we will merge this into the rest of our repos with some minor changes.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Reporter checklist
+      options:
+        - label: I am able to reproduce the bug with the latest version of the code present in this repository
+          required: true
+        - label: I am able to install and run a development version of the code present in this repository
+          required: true
+        - label: I have gathered debug logs for the moment when the problem happened
```
Comment (kang-makes) [inline .github/ISSUE_TEMPLATE/bug_report.yml:25]: Could we add a link here to show people how to change the log level?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -1,10 +1,8 @@
 blank_issues_enabled: false
-
 contact_links:
   - name: Open a support ticket
     url: https://support.newrelic.com/
-    about: Get help installing, deploying or troubleshooting New Relic products. Our support team is here to help!
-
-  - name: Request a feature
```
Comment (gsanchezgavier) [inline .github/ISSUE_TEMPLATE/config.yml:8]: I think not having a FR path will make users to submit FR as Bugs which will make more difficult to track them or will add some friction to the communication when this happen.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    validations:
+      required: true
+  - type: input
+    id: env-service
+    attributes:
+      label: Affected service
+      description: Which service were you trying to monitor?
+      placeholder: MySQL 8, Postgres 12, NGINX 1.21, Kubernetes 1.22...
+    validations:
+      required: true
```
Comment (kang-makes) [inline .github/ISSUE_TEMPLATE/bug_report.yml:56]: Is this needed? They are going to open a bug in each integration. This could only be valid in Kubernetes that we monitor more than one service. Am I right?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+      description: Configuration file used to reproduce the problem. This will be automatically formatted as YAML, so no need for backticks.
+      render: yaml
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: shell
```
Comment (kang-makes) [inline .github/ISSUE_TEMPLATE/bug_report.yml:92]: I would no render the logs because weird things will happen with `"` and `'`. But I still would say to the user that this is going to be automatically formatted so they would not add backticks.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 5]
```diff
@@ -1,10 +1,8 @@
...
-    url: https://support.newrelic.com/
-    about: Request a missing feature in the New Relic experience.
```
Comment (kang-makes) [inline .github/ISSUE_TEMPLATE/config.yml:10]: I am with @gsanchezgavier in adding a feature request if it is possible so people do not open feature requests as bugs.

**Review by kang-makes — APPROVED** [review]
Awesome work. I just left NITs.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+        If you are not a developer, or if you are having trouble identifying, isolating, or reproducing an issue, please reach out to our support team, which is better equipped to help you in that regard:
+        Visit https://support.newrelic.com/
+  - type: checkboxes
+    id: checks
+    attributes:
+      label: Reporter checklist
+      options:
+        - label: I am able to reproduce the bug with the latest version of the code present in this repository
+          required: true
+        - label: I am able to install and run a development version of the code present in this repository
```
Comment (paologallinaharbur) [inline .github/ISSUE_TEMPLATE/bug_report.yml:23]: This could be a bit scary (but I imagine that is intended to be so)

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (paologallinaharbur) [reply .github/ISSUE_TEMPLATE/bug_report.yml:25]: I would rephrase the sentence "I have gathered debug logs for the moment when the problem happened" sounds weird

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+      placeholder: |
+        1. Set up ...
+        2. Configure ...
+    validations:
+      required: true
+  - type: textarea
+    id: configuration
+    attributes:
+      label: Configuration file
+      description: Configuration file used to reproduce the problem. This will be automatically formatted as YAML, so no need for backticks.
```
Comment (paologallinaharbur) [inline .github/ISSUE_TEMPLATE/bug_report.yml:83]: I would add a note to sanitize the config/output, being a form they could expect (somehow) to have config hidden to public

**Review by paologallinaharbur — APPROVED** [review]
Some nits, but looks good
